include/rocksdb/options.h

```diff
@@ -1024,6 +1024,9 @@ struct DBOptions {
   // DEFAULT: false
   bool fail_if_options_file_error = false;
 
+  // If false, we won't use options file.
```

Comment:
@igorcanadi let's add another line of comment like this
```
  // DEFAULT: false
```
to be consistent with other options in this file